chef/cookbooks/suse-manager-client/recipes/default.rb

```diff
@@ -17,8 +17,10 @@
 # limitations under the License.
 #
 
+return unless node[:platform_family] == "suse"
```

Comment:
We don't, right now. Maybe just raise an exception for the RHEL case so that people who try it will fix it instead of assuming it works fine.